ci/pipelines/cf-for-k8s.yml

```diff
@@ -1442,6 +1443,235 @@ jobs:
+      platform: linux
+      run:
+        args:
+        - -ec
+        - |
+          source cf-for-k8s-ci/ci/helpers/gke.sh
+
+          cluster_name="$(cat tf-vars/env-name.txt)"
+          gcloud_auth "${cluster_name}"
+
+          kubectl create namespace external-db
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm install -n external-db --wait postgresql bitnami/postgresql
+
+          CCDB_USERNAME="capi_user"
+          CCDB_PASSWORD="$(openssl rand -base64 32)"
+          CCDB_NAME="capi_db"
+          UAADB_USERNAME="uaa_user"
+          UAADB_PASSWORD="$(openssl rand -base64 32)"
+          UAADB_NAME="uaa_db"
+
+          cat > /tmp/setup_db.sql <<EOT
+          CREATE DATABASE ${CCDB_NAME};
+          CREATE ROLE ${CCDB_USERNAME} LOGIN PASSWORD '${CCDB_PASSWORD}';
+          CREATE DATABASE ${UAADB_NAME};
+          CREATE ROLE ${UAADB_USERNAME} LOGIN PASSWORD '${UAADB_PASSWORD}';
+          \c ${CCDB_NAME};
+          CREATE EXTENSION citext;
+          \c ${UAADB_NAME};
+          CREATE EXTENSION citext;
+          EOT
+
+          export POSTGRES_PASSWORD=$(kubectl get secret -n external-db postgresql -o jsonpath="{.data.postgresql-password}" | base64 -d)
+          kubectl exec -n external-db statefulset/postgresql-postgresql -i -- psql "postgresql://postgres:$POSTGRES_PASSWORD@localhost:5432/postgres" < /tmp/setup_db.sql
+
+          cat > db-metadata/db-values.yaml <<EOT
+          #@data/values
+          ---
+          capi:
+            database:
+              adapter: postgres
+              host: postgresql.external-db.svc.cluster.local
+              port: 5432
+              user: $CCDB_USERNAME
+              password: $CCDB_PASSWORD
+              name: $CCDB_NAME
+
+          uaa:
+            database:
+              adapter: postgresql
+              host: postgresql.external-db.svc.cluster.local
+              port: 5432
+              user: $UAADB_USERNAME
+              password: $UAADB_PASSWORD
+              name: $UAADB_NAME
+          EOT
+        path: /bin/bash
+    input_mapping:
+      pool-lock: ready-pool
+  - do:
+    - task: install-cf-develop
+      params:
+        GCP_SERVICE_ACCOUNT_JSON: ((ci_k8s_gcp_service_account_json))
+        GCP_PROJECT_NAME: ((ci_k8s_gcp_project_name))
+        GCP_PROJECT_ZONE: ((ci_k8s_gcp_project_zone))
+        DOMAIN: cf-for-k8s.relint.rocks
+        USE_EXTERNAL_APP_REGISTRY: true
+        APP_REGISTRY_HOSTNAME: https://index.docker.io/v1/
+        APP_REGISTRY_REPOSITORY_PREFIX: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_USERNAME: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_PASSWORD: ((cf_for_k8s_private_dockerhub.password))
+        USE_EXTERNAL_DB: "true"
+      file: cf-for-k8s-ci/ci/tasks/install-cf-on-gke/task.yml
+      input_mapping:
+        cf-for-k8s: cf-for-k8s-develop
+
+  - task: push-test-app
+    attempts: 3
```

Comment:
please remove these, they're no longer desirable

Reply:
Which ones exactly?

Reply:
I suppose @jamespollard8  is referring to the multiple 'attempts'?

Reply:
Yes, the multiple attempts have been removed since the PR was initially opened. 👍